packages/react-vapor/src/components/flatSelect/tests/FlatSelectSelectors.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import { flatSelectInitialState, IFlatSelectState } from '../FlatSelectReducers';
+import { FlatSelectSelectors } from '../FlatSelectSelectors';
+
+describe('FlatSelectSelectors', () => {
+    describe('getInput', () => {
+        it('Should return the default flatSelect state', () => {

```

Comment:
I'd change the name to be more specific

```suggestion
        it('should return the default flatSelect state when the flatSelect is not in the store', () => {
```